packages/react/package.json

```diff
@@ -9,7 +9,7 @@
 		"@floating-ui/react-dom": "^2.0.8",
 		"@react-spring/web": "^9.4.5",
 		"date-fns": "^2.28.0",
-		"downshift": "^8.1.0",
+		"downshift": "^9.0.8",
```

Comment:
What did the upgrade fix?

Reply:
[It adds the getA11yStatusMessage function to the useMultipleSelection hook](https://github.com/downshift-js/downshift/blob/59d3498faafd3c3b4832a20dab7485502ce67284/src/hooks/MIGRATION_V9.md?plain=1#L58-L59). We were using the `itemToString` function which was meant to allow us to select the label from the object but it wasn't working in this way. It was being called but ultimately the component was not announcing the returned string, just the object.toString.

Reply:
OK cool. Doesn't look like the upgrade has any other major changes, so shouldn't need to test the normal combobox much…

Reply:
Yeah, I've tested it all manually on Chrome desktop and iOS Safari.